manifests/a/AnalogDevices/LTspice/17.1.15.0/AnalogDevices.LTspice.installer.yaml

```
@@ -0,0 +1,19 @@
+# Automatically updated by the winget bot at 2023/Nov/10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AnalogDevices.LTspice
+PackageVersion: 17.1.15.0
+InstallerLocale: en-US
+InstallerType: msi
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: LTspice
+  Publisher: Analog Devices Inc.
+  DisplayVersion: 17.1.10.0

```

```suggestion
  DisplayVersion: 17.1.15.0
```

Change `AppsAndFeaturesEntries[*].DisplayVersion` to match `PackageVersion`.

---

DisplayVersion should not be equal to PackageVersion. DisplayVersion should be removed altogether from this manifest.

The bot cannot accept/resolve changes, would you be willing to create a PR for it?

---

Okay, I'll make it.